Compiler/General.py

```diff
@@ -606,7 +610,7 @@ def get_unary_postfix_op_code(token, offset_to_variable):
     raise NotImplementedError
 
 
-def get_op_between_literals_code(op_token):
+def get_op_between_literals_code(op_token, left_token=None, right_token=None):
```

Comment:
Please remove unused left_token

Reply:
I was thinking of trying to implement more compiler optimizations but i guess that i can add the left_token parameters in that pull request